### ISSUE TITLE
Is NLS_NCHAR_CHARACTERSET being set to UTF8 the expected behaviour with this image?

### ISSUE THREAD

**Issue by rmoff** [oracle/docker-images#311]
I was trying to use this image with 12.2 for Property Graph, but it was failing during `opg_apis.create_pg`: 

```
Error report -
ORA-01450: maximum key length (6398) exceeded
ORA-06512: at "MDSYS.OPG_APIS", line 1444
ORA-06512: at "MDSYS.OPG_APIS", line 1116
ORA-06512: at "MDSYS.OPG_APIS", line 1213
ORA-06512: at "MDSYS.OPG_APIS", line 464
ORA-06512: at line 1
01450. 00000 -  "maximum key length (%s) exceeded"
*Cause:    
*Action:
```

On digging into it we found that `NLS_NCHAR_CHARACTERSET` was `UTF8`. I invoked `docker run` with no `ORACLE_CHARACTERSET` specified, so believe it used the default of `AL32UTF8`. 

Is `NLS_NCHAR_CHARACTERSET` being set to `UTF8` the expected behaviour with this image? 

```
SQL> select * from V$NLS_PARAMETERS;

PARAMETER                        VALUE                                CON_ID
-------------------------------- -------------------------------- ----------
NLS_LANGUAGE                     AMERICAN                                  3
NLS_TERRITORY                    AMERICA                                   3
NLS_CURRENCY                     $                                         3
NLS_ISO_CURRENCY                 AMERICA                                   3
NLS_NUMERIC_CHARACTERS           .,                                        3
NLS_CALENDAR                     GREGORIAN                                 3
NLS_DATE_FORMAT                  DD-MON-RR                                 3
NLS_DATE_LANGUAGE                AMERICAN                                  3
NLS_CHARACTERSET                 AL32UTF8                                  3
NLS_SORT                         BINARY                                    3
NLS_TIME_FORMAT                  HH.MI.SSXFF AM                            3
NLS_TIMESTAMP_FORMAT             DD-MON-RR HH.MI.SSXFF AM                  3
NLS_TIME_TZ_FORMAT               HH.MI.SSXFF AM TZR                        3
NLS_TIMESTAMP_TZ_FORMAT          DD-MON-RR HH.MI.SSXFF AM TZR              3
NLS_DUAL_CURRENCY                $                                         3
NLS_NCHAR_CHARACTERSET           UTF8                                      3
NLS_COMP                         BINARY                                    3
NLS_LENGTH_SEMANTICS             BYTE                                      3
NLS_NCHAR_CONV_EXCP              FALSE                                     3

19 rows selected.
```

**Comment by gvenzl** [comment]
Thanks @rmoff, you did in fact spot a bug with the dbca response file. I'm on it, shouldn't take too long!

**Comment by gvenzl** [comment]
Fixed with #312

**Comment by rmoff** [comment]
Awesome, thanks @gvenzl 👍
